modules/contentpool_normalization/src/EventSubscriber/ContentpoolNormalizationEventSubscriber.php

```diff
@@ -45,6 +45,11 @@ public function onAlterContentData(ReplicationContentDataAlterEvent $event) {
 
     if ($entity->getEntityTypeId() == 'node' && $entity->bundle() == 'article') {
       foreach ($language_keys as $key) {
+        $paragraph_data = [];
+        foreach ($entity->field_paragraphs->getValue() as $delta => $value) {
+          $paragraph_data[$delta] = $entity->field_paragraphs->get($delta)->entity->toArray();
+        }
+        $normalized[$key]['field_data'][] = $paragraph_data;
```

Comment:
The content is good, but let's call it as what is: field_data -> field_paragraphs_data

Reply:
oh, no you are right. the concept says generic data.